masterlist.yaml

```
@@ -2658,19 +2658,33 @@ plugins:
       - Graphics
       - NoMerge
 
-  - name: 'Improved Chests.esp'
+  - name: 'Improved Chests( \(OOO Compatible\))?\.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/51815/'
         name: 'Improved Chests'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Improved Chests ESP' ]
+        condition: 'many("Improved Chests( \(OOO Compatible\))?\.esp")'
     tag:
       - Graphics
       - Names
-      - Sound
+  - name: 'Improved Chests.esp'
+    msg:
+      - <<: *useVersion
+        subs: [ 'Oscuro''s Oblivion Overhaul' ]
+        condition: 'active("Oscuro''s_Oblivion_Overhaul.esm") and active("Oscuro''s_Oblivion_Overhaul.esp")'
```

Since `Oscuro's_Oblivion_Overhaul.esp` has `Oscuro's_Oblivion_Overhaul.esm` as a master, this condition can be simplified:

```suggestion
        condition: 'active("Oscuro''s_Oblivion_Overhaul.esp")'
```